nomenclature/definitions/variable/fuel_types.yaml

```diff
@@ -0,0 +1,80 @@
+# This list of fuels will replace any occurrence of `<Fuel>`
+# in a variable-definition yaml file
+
+# All fuels that have the attribute "ccs: True" have two sub-categories
+# "w/ CCS": with a CO2 capture component
+# "w/o CCS": with freely vented CO2 emissions
+
+<Fuel>:
+  Biomass:
+    description: Purpose-grown biomass, crop residues, forest industry waste,
+                 municipal solid waste
+    ccs: True
+
+  Coal:
+    description: Coal (including hard coal and lignite)
+    ccs: True
+
+  Fossil:
+    description: All fossil fuels (crude oil, coal, natural gas)
+    ccs: True
+
+  Gas:
+    description: Natural gas (including methane from biomass or hydrogenation)
+    ccs: True
+
+  Gas|Fossil:
+    description: Natural gas (excluding methane from biomass or hydrogenation)
+    ccs: True
+
+  Gas|Biomethane:
+    description: Biomethane (upgraded biogas)
+    ccs: True
+
+  Gas|Synthetic Methane:
+    description: Synthetic methane (methanation of hydrogen)
+    ccs: True
+
+  Oil:
+    description: Refined liquid oil products
+    ccs: True
+
+  Nuclear:
+    description: Nuclear energy
+
+  Hydrogen:
+    description: Hydrogen
+
+  Non-Biomass Renewables:
+    description: Hydro, wind, solar, geothermal, ocean, and other renewable
+                 sources excluding bioenergy
+
+  Hydro:
+    description: Hydropower
+
```

Comment:
```suggestion
Hydro|Run of River:
description: Hydropower produced by run-of-river power plants (ie. without  storage capacity). 

Hydro|Pumped Storage:
description: Hydropower systems with short-term storage capacity and including pumping turbines. This hydropower plant has two water reservoirs at different elevations: one at the upper level and one at the lower level. It can generate power (discharge) as water moves down through a turbine; however, it draws power as it pumps water (recharge) to the reservoir at the upper level.

Hydro|Reservoir:
description: Hydropower systems with seasonal storage capacity (it can also include) pumping turbines. This hydropower plant has two water reservoirs at different elevations: one at the upper level and one at the lower level. It can generate power (discharge) as water moves down through a turbine; however, it draws power as it pumps water (recharge) to the reservoir at the upper level.
```

Reply:
thanks for the suggestion @sandrinecharousset and @erikfilias. If I understand this correctly, you propose:

 - pumped storage: short-term storage
 - reservoir: dams with seasonal storage that can also include pumps - so this would also be pumped storage?

I'm worried that this can lead to quite some ambiguities. As an alternative, I'd suggest to have
 - Hydro|Run of River
 - Hydro|Reservoir
 - Hydro|Reservoir|Pumped Storage
(and maybe option for reservoirs that just have natural inflows, ie. no pumping)

Reply:
Hi @danielhuppmann 
What I wanted to do is separate seasonal storage from short-ter storage; Both may include pumping facilities (or not); this is quite usual in electricity management : in Entso-e transparency platform, you can find the list of technologies, for hydro it includes Hydro Pumped Storage and Hydro Water reservoir (which is why we chose those names)(and also Hydro Run-of-river and poundage)
Could we keep the names and include in the description that it is entso-e nomenclature? (in that case maybe we could stick to the exact entso-e nomenclature:
- Hydro|Run of river and poundage
- Hydro|Water reservoir
- Hydro|Pumped storage

Reply:
@danielhuppmann , I find ambiguous to have Reservoir and Reservoir|Pumped Storage because Pumped Storage do not have a real reservoir (only smaller ones which are not called the same name.... but I only know the french names sorry....)

Reply:
@danielhuppmann  an other alternative could be to include the reference to the entso-e nomenclature in the definition and to use
Hydro|Seasonal Storage   (entso-e: Hydro Water reservoir)
Hydro|Short-term storage (entso-e: Hydro Pumped storage)

Reply:
> @danielhuppmann an other alternative could be to include the reference to the entso-e nomenclature in the definition and to use
> Hydro|Seasonal Storage (entso-e: Hydro Water reservoir)
> Hydro|Short-term storage (entso-e: Hydro Pumped storage)

I agree with it. And, also with `Hydro|Run of River and poundage`.

Reply:
Thank you for the clarification about ENTSO-E terminology, I'll add the proposal and a note referring to ENTSO-E